src/components/Header/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import HeaderLogin from '../HeaderLogin';
+import HeaderToggle from '../HeaderToggle';
+import HeaderSearch from '../HeaderSearch';
+import HeaderNav from '../HeaderNav';
+
+const msgPlaceholder = [
+  'What can we help you find?',
+  'Today, you are wanna watch...',
+  'If you think about it, just look it up',
+];
+
+function getRand(max) {
```

Comment:
Perhaps a better name for this would be `getRandomPlaceHolder` and return the random placeholder text already.